Source/Core/GroundPolylineGeometry.js

```diff
@@ -457,6 +457,11 @@ define([
         cartographics = arrayRemoveDuplicates(cartographics, Cartographic.equalsEpsilon);
         cartographicsLength = cartographics.length;
 
+        // In order to support external data that may have errors we treat this as an empty geometry.
```

Comment:
We generally don't use inline comments unless something is truly non-obvious.  It's easy for the inline comments to get out of sync with the code when we make changes.
We honestly should probably remove most of the comments in the files you added.